packages/core/contracts/insured-bridge/BridgeAdmin.sol

```diff
@@ -231,14 +235,19 @@ contract BridgeAdmin is BridgeAdminInterface, Ownable, Lockable {
     }
 
     /**************************************
-     *        VIEW FUNCTIONS              *
+     *           VIEW FUNCTIONS           *
      **************************************/
     function depositContracts(uint256 _chainId) external view override returns (DepositUtilityContracts memory) {
         return _depositContracts[_chainId];
     }
 
-    function whitelistedTokens(address _l1Token) external view override returns (L1TokenRelationships memory) {
-        return _whitelistedTokens[_l1Token];
+    function whitelistedTokens(address _l1Token, uint256 chainId)
+        external
+        view
+        override
+        returns (address l2Token, address bridgePool)
+    {
+        return (_whitelistedTokens[_l1Token].l2Tokens[chainId], _whitelistedTokens[_l1Token].bridgePool);
```

Comment:
You can't return `L1TokenRelationships`?

Reply:
nope. this does not work. you get this:

<img width="660" alt="image" src="https://user-images.githubusercontent.com/12886084/136047738-a6af701c-b81e-4148-81b6-1d23c946f69a.png">